metadata-service/auth-ranger-impl/src/main/java/com/datahub/authorization/ranger/AuthorizerConfig.java

```diff
@@ -0,0 +1,30 @@
+package com.datahub.authorization.ranger;
+
+import java.util.Map;
+import java.util.Optional;
+
+
+public class AuthorizerConfig {
+  private final Map<String, Object> authorizerConfigMap;
+
+  public AuthorizerConfig(Map<String, Object> authorizerConfigMap) {
```

Comment:
We should prob verify that the config map contains required values when it is created.

Reply:
I have converted the Map to Pojo for validation